ui/app/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -739,7 +748,7 @@ export default class ConfirmTransactionBase extends Component {
       if (transactionCategory) {
         functionType = t(transactionCategory) || transactionCategory
       } else {
-        functionType = t('contractInteraction')
+        functionType = t(TRANSACTION_CATEGORIES.CONTRACT_INTERACTION)
```

Comment:
This one should be removed, will do before I move out of draft.

Reply:
At least I think so..

Reply:
changed this back, but leaving unresolved for other thoughts

Reply:
I think you made the right call